internal/domain/review_suggestion.go

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import (
+	"github.com/invopop/jsonschema"
+)
+
+type ReviewSuggestion struct {
+	Comment  string `json:"comment"`
```

Comment:
The function signature `func GenerateSchema[T any]() interface{}` could be more specific by returning a concrete type like `*jsonschema.Schema`. This will enhance type safety and clarity for users of the function, making it clear that it returns a schema rather than a generic interface{}.